### DISCUSSION TITLE
Can't get Docker to run

### DISCUSSION THREAD

**Discussion by sasori7** [SWivid/F5-TTS#573] [General]
Has anyone been able to run this in docker?  I get this error and nothing runs:
WARNING: The NVIDIA Driver was not detected.
I run XTTS without any problem (but not within Docker.)

**Comment by larryjameshenry** [comment]
If you have the Nvidia container toolkit installed and working.  I'm using windows with an updated WSL, so the driver is available and all I did was:

docker pull ghcr.io/swivid/f5-tts:main
docker run --runtime=nvidia --gpus all -it ghcr.io/swivid/f5-tts:main
